### ISSUE TITLE
Allow aria-label attributes to be easily changed/translated

### ISSUE THREAD

**Issue by henrahmagix** [insites/cookieconsent#237]
I used this (wonderful!) tool for a French site recently, and have learnt that the `aria-label` attributes should be translated because they're free text. From https://webaim.org/discussion/mail_thread?thread=6620#post2:
>The aria-label attribute, or the alt or title attribute, are different: 
by definition, their values are not identifiers but "free" text, 
basically expected to be in a human language understandable to the user 
(and suitable in the context of the page). Thus, **translating them is 
essential** and quite comparable to translating normal text content...

However this tool doesn't make it easy to do so: the templates have to be copied completely from the source and then altered.

Could the `aria-label` messages be moved to the `content` object so they can be easily translated, as #214 suggests with the visible content?

**Comment by Enkrod** [comment]
I don't know if this actually presents a solution to your problem, but it certainly helped me.

As you know you can copy the element templates from the source and alter them. But you can also just introduce new variables to be inserted, like this:

```
window.cookieconsent.initialise({
    elements: {
        allow: '<a aria-label="{{newlyIntroducedAallowLabel}}" role=button tabindex="0"  class="cc-btn cc-allow">{{allow}}</a>'
    },
    content: {
        allow: "Ok",
        newlyIntroducedAallowLabel: "Allow cookies"
    }
});
```
Granted, it would be better if this was actually already in the source (if only to get smaller JS) but I think that's pretty nifty and easy to use already.

**Comment by alexmorleyfinch** [comment]
They could be yes but it's probably better to just re-write the options. (It sucks I know). It's hard to make the input HTML configurable for everybody, and the whole system kind of seems overkill in retrospect now.

If I did it again I'd probably just pass a single `html` option that would have to be re-written for any kind of custom work. Would make the learning curve a lot easier, and would move the issue of translation etc onto the user.
